<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptsPage</class>
 <widget class="QWidget" name="ReceiptsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
}

QDialog {
    background-color: white;
    border-radius: 10px;
    padding: 10px;
}

QLabel {
    font: 14px &quot;Arial&quot;;
    color: black;
    padding: 2px 0;
}

QLineEdit, QComboBox {
    border: 1px solid rgb(200, 200, 200);
    border-radius: 5px;
    padding: 5px;
    background-color: rgb(245, 245, 245);
    font: 14px &quot;Arial&quot;;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_ReceiptsTitle">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: black;
    font: bold 35px &quot;Palatino Linotype&quot;;
    padding: 10px;
}
</string>
     </property>
     <property name="text">
      <string>Receipts</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="button_BackToDashboard">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(28, 130, 255);
    border: none;
    border-radius: 15px;
    color: white;
    padding: 10px 20px;
    font: bold 15px &quot;Arial&quot;;
}

QPushButton:hover {
    background-color: rgb(33, 145, 255);
}

QPushButton:pressed {
    background-color: rgb(23, 110, 230);
}
</string>
     </property>
     <property name="text">
      <string>Back to Dashboard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableWidget" name="table_Receipts">
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    border-radius: 10px;
    background-color: white;
    gridline-color: rgb(220, 220, 220);
    font: 14px &quot;Arial&quot;;
}

QHeaderView::section {
    background-color: rgb(240, 240, 240); 
    color: black; 
    font: bold 14px &quot;Arial&quot;;
    padding: 4px;
    border: 1px solid rgb(200, 200, 200);
}

QTableWidget::item:selected {
    background-color: rgb(235, 245, 255); 
    color: black;
}

QComboBox {
    border: 1px solid rgb(200, 200, 200); 
    border-radius: 5px; 
    padding: 4px 8px;
    background-color: white; 
    font: 14px &quot;Arial&quot;; 
}

QComboBox::drop-down {
    border: none;
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px; 
}

QComboBox::down-arrow {
    image: url(:/icons/down-arrow.png); 
    width: 10px;
    height: 10px;
}

QComboBox QAbstractItemView {
    border: 1px solid rgb(200, 200, 200); 
    border-radius: 5px; 
    background-color: white;
    font: 14px &quot;Arial&quot;; 
    selection-background-color: rgb(220, 220, 220); 
    selection-color: black;
}

</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>220</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Store</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_UploadReceipt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(28, 130, 255);
    border: none;
    border-radius: 15px;
    color: white;
    padding: 10px 20px;
    font: bold 15px &quot;Arial&quot;;
}

QPushButton:hover {
    background-color: rgb(33, 145, 255);
}

QPushButton:pressed {
    background-color: rgb(23, 110, 230);
}</string>
       </property>
       <property name="text">
        <string>Upload Receipt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_EditReceipt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(28, 130, 255);
    border: none;
    border-radius: 15px;
    color: white;
    padding: 10px 20px;
    font: bold 15px &quot;Arial&quot;;
}

QPushButton:hover {
    background-color: rgb(33, 145, 255);
}

QPushButton:pressed {
    background-color: rgb(23, 110, 230);
}</string>
       </property>
       <property name="text">
        <string>Edit Receipt</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
